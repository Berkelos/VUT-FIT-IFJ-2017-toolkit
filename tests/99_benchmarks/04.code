' complex expression

scope
	dim a as integer = 74
	dim b as double = 745.745
	
	dim c as double

	c = (a * a) + b / b - b * a + (b * (a * a + b * a + 38 * 7 + 2 * 5 * b) - b * 78 * 96 - b * a + (b + b + b + b + b + b))
	print c;
end scope