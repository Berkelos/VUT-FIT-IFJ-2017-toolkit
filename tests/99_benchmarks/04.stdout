-44485.28081223562
 114.4530138595595